src/common/serialization.tcc

```diff
@@ -76,7 +76,7 @@ T reserialize(const T &obj)
 template<typename T>
 std::ostream& operator<<(std::ostream& out, const std::vector<T> &v)
 {
-    static_assert(!std::is_same<T, bool>::value, "this does not work for std::vector<bool>");
+    static_assert(!(std::is_same<T, bool>::value), "this does not work for std::vector<bool>");
```

Comment:
I've seen this before. It depends on the compiler, but the angle brackets for the templates screw with the parser.